packages/botonic-pulumi/src/pulumi-runner.ts

```diff
@@ -222,9 +235,38 @@ export class PulumiRunner {
       this.replaceWithWebSocketUrl(websocketReplacementUrl)
     }
     const frontendResults = await this.runStack('frontend')
+    if (frontendResults && this.updatedBucketObjects.length > 0) {
+      await this.doInvalidateUpdatedFiles(
+        frontendResults,
+        this.updatedBucketObjects
+      )
+    }
     return
   }
 
+  private async doInvalidateUpdatedFiles(
+    updateResults: UpResult,
+    updatedBucketObjects: string[]
+  ): Promise<void> {
+    try {
+      const cacheInvalidator = new CacheInvalidator({
+        region: this.projectConfig.region,
+        profile: this.projectConfig.profile,
+        accessKey: this.projectConfig.accessKey,
+        secretKey: this.projectConfig.secretKey,
+        token: this.projectConfig.token,
+      })
```

Comment:
Could `this.projectConfig` be passed directly? All the attributes of the passed object have the same names.

Reply:
yes, in fact I wanted to be explicit because projectConfig comes with other attributes which doesn't match with the `AWSCredentials` type that CacheInvalidator is expecting. But as it is a subset of the config, I have passed it like `this.projectConfig as AWSCredentials`.